src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    sum = days * 40;
+  }
+
+  if (days >= 3) {
+    sum = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    sum = days * 40 - 50;
+  }
```

Comment:
In this case, when the function is running and the number is greater than = 7, the variable will first be counted in condition (days >= 3) and then overwritten in condition (days >= 7). Perhaps this process could be more optimized?